common/config/environment.js

```diff
@@ -24,3 +24,8 @@ export const clientTokens = isProduction
 export const apiUrl = isProduction
   ? 'https://backend.k8s.operationcode.org'
   : 'https://backend-staging.k8s.operationcode.org';
+
+export const slackApiUrl = 'https://slack.com/api';
```

Comment:
I don't anticipate using the Slack API for anything else in the near future. Can we simplify this to one URL?

```suggestion
export const slackMembersAPIUrl = 'https://slack.com/api/conversations.members';
```

Reply:
👍